### PR TITLE
Add community section

### LINKED ISSUE #928: Add community / help section
We should add a section about how to get help for users, linking to GitHub discussions, etc

### PR DESCRIPTION
**Pull request by pascalberger**
Add community section with information about discussion channels and how to get help.

In subsequent PRs more content can be moved to the community section (Resources, Contribution Info, Thanks page)

Fixes #928

### CODE REVIEW

**Comment by pascalberger** [comment]
Once this is merged we also need to add the new pages to search index

File: input/community/getting-help.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[GitHub Discussions](https://github.com/cake-build/cake/discussions) is the place to get in touch with the community if you get stuck with Cake.
+
+<a class="btn btn-primary btn-lg" href="https://github.com/cake-build/cake/discussions?discussions_q=category%3AQ%26A" target="_blank" role="button">
+    <i class="fa fa-search"></i> Browse questions & answers
+</a>
+<a class="btn btn-primary btn-lg" href="https://github.com/cake-build/cake/discussions/new?category_id=7859135" target="_blank" role="button">
+    <i class="fa fa-life-ring"></i> Create a new question
+</a>
+
+# StackOverflow
```
Comment (gep13) [inline input/community/getting-help.md:17]:
```suggestion
# Stack Overflow
```

File: input/community/getting-help.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+There's a [StackOverflow tag](https://stackoverflow.com/questions/tagged/cakebuild) which you can use when posting messages about Cake.
```
Comment (gep13) [inline input/community/getting-help.md:19]:
```suggestion
There's a [Stack Overflow tag](https://stackoverflow.com/questions/tagged/cakebuild) which you can use when posting messages about Cake.
```

**Review by gep13 — CHANGES_REQUESTED** [review]
Just a couple minor things, but otherwise, ready to go.

**Comment by pascalberger** [comment]
@gep13 Ready for another review

**Review by gep13 — APPROVED** [review]
LGTM

**Pull request merged**